source/mainnet/net/guides/become-baker.rst

```diff
@@ -129,11 +124,10 @@ The following arguments are optional. If no selection is made, earnings are rest
 .. Warning::
    Do not stake all of your funds or you will not have enough funds to cover transaction fees.
 
-To start the node with these baker keys and bake blocks, you
-first need to shut down the current running node. To do this, either press ``Ctrl + C`` on the terminal where the node is running or use the
-``concordium-node-stop`` executable.
-
-When you've placed the file in the appropriate directory, which is what you did you did in the previous command when you specified the output file, start the node again using ``concordium-node``. The node will automatically start baking when the baker is included in the bakers for the current epoch.
+To start the node with these baker keys and bake blocks configure the node to
+use the baker keys, and restart it., you first need to shut down the current
+running node. The node will automatically start baking when the baker is
```

Comment:
Given that we already write node needs to be restarted, is this intentional?
", you first need to shut down the current running node."

Reply:
No, I messed up the editing here.